remsfal-core/src/main/java/de/remsfal/core/json/project/TaskItemJson.java

```diff
@@ -39,7 +39,8 @@ public static TaskItemJson valueOf(final TaskModel model) {
             .name(model.getTitle())
             .title(model.getTitle())
             .status(model.getStatus())
-            .owner(model.getOwnerId())
```

Comment:
Ich konnte den Ursprung der NullPointerException zurückverfolgen wenn ich diesen part auskommentiere und hardwire erhallte ich keinen Nullpointer Fehler Mehr allerdings kann ich nicht nachvollziehen wo in dieser getOwnerId Methode der Fehler herkommt